packages/contracts-bedrock/tasks/update-dynamic-oracle-config.ts

```diff
@@ -0,0 +1,98 @@
+import readline from 'readline'
+
+import { task, types } from 'hardhat/config'
+import { ethers, Wallet } from 'ethers'
+
+import { getContractsFromArtifacts } from '../src/deploy-utils'
+
+task('update-dynamic-oracle-config', 'Updates the dynamic oracle config.')
```

Comment:
Do we want a `--force` option to skip the prompt?

Reply:
I think the prompt is important. We'll never run this in CI, and it's important that these values are correct.

Reply:
ie, running this script as part of our contract deployment to get test coverage. I think you can do `hre.run('update-dynamic-oracle-config')` from within another hh script to run it

Reply:
In particular i mean the deployment script that deploys the system, this could replace the part of the script that sets the dynamic gas config

Reply:
Ah I see. Fair enough. Can update.